CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
+ - Fixed auto-contribution table being corrupted when changing "Minimum page time before logging a visit" via Brave Rewards settings. ([#3134](https://github.com/brave/brave-browser/issues/3134))
+ - Fixed publishers being removed from the auto-contribution table when "Minimum page time before logging a visit" is changed and Brave is upgraded or restarted. ([#3162](https://github.com/brave/brave-browser/issues/3162))
+ - Fixed Reward Panel not always displaying Welcome screen on first click which caused issues creating wallets. ([#2962](https://github.com/brave/brave-browser/issues/2962))
+ - Fixed several instances of incorrect crypto addresses being displayed in Brave Rewards. ([#2172](https://github.com/brave/brave-browser/issues/2172))
+ - Fixed weight for each publisher within auto-contribution not being calculated correctly. ([#3163](https://github.com/brave/brave-browser/issues/3163))
+ - Fixed Widevine notifications not appearing under YouTube and Amazon Prime when viewing premium content and Widevine is not installed. ([#1985](https://github.com/brave/brave-browser/issues/1985))
+ - Fixed PDFs being opened in Brave even though "Download PDF files instead of automatically opening them in Brave" has been enabled in Brave settings. ([#1531](https://github.com/brave/brave-browser/issues/1531))
+ - Fixed disabling auto-contribution also disabled YouTube tipping. ([#2084](https://github.com/brave/brave-browser/issues/2084))
+ - Fixed notifications under Brave Panel not being dismissible in certain instances. ([#2883](https://github.com/brave/brave-browser/issues/2883))
+ - Fixed several instances of Rewards Panel not displaying YouTube publishers due to different URLs. ([#2820](https://github.com/brave/brave-browser/issues/2820))
+ - Fixed pending contributions to unverified publishers not being dynamically updated under Brave Rewards. ([#2890](https://github.com/brave/brave-browser/issues/2890))
+ - Fixed Brave Rewards summary sometimes displaying incorrect tip amounts. ([#2332](https://github.com/brave/brave-browser/issues/2332))
+ - Fixed non-verified publishers not being displayed in Brave Panel when unchecking "Allow contributions to non-verified sites" in Brave Rewards settings. ([#2653](https://github.com/brave/brave-browser/issues/2653))
+ - Fixed "Next Contribution Date" under Brave Rewards displaying incorrect dates in some instances. ([#2206](https://github.com/brave/brave-browser/issues/2206))
+ - Fixed non-verified sites being added into auto-contribution table when "Allow contributions to non-verified sites" has been disabled. ([#2111](https://github.com/brave/brave-browser/issues/2111))
+ - Fixed "Send My Tip" button under Brave Panel not displaying cursor. ([#2744](https://github.com/brave/brave-browser/issues/2744))
+ - Fixed switching themes within Brave settings not updating the DevTools theme. ([#784](https://github.com/brave/brave-browser/issues/784))
```

Comment:
Suggested text: Added dark theme support for Developer Tools. 
Could also bump this towards the top, maybe under or before Improved Welcome experience.